projects/plugins/crm/tests/php/bootstrap.php

```diff
@@ -14,3 +14,17 @@
  * Load WorDBless.
  */
 \WorDBless\Load::load();
+
+/**
+ * Load all feature flags, so they will be testable.
+ */
+add_filter( 'jetpack_crm_feature_flag_api_v4', '__return_true' );
+
+/**
+ * Load Jetpack CRM.
+ *
+ * Not all code is automatically loaded any we depend on a lot of global
+ * variables, so the easiest path forward (for now at least) is to just
+ * load the core plugin file so everything is initiated.
+ */
+require_once __DIR__ . '/../../ZeroBSCRM.php';
```

Comment:
Note that this won't always work depending on the setup; we should avoid hard-coding path traversal.

Reply:
>we should avoid hard-coding path traversal
That's my default approach as well, but we can't use e.g. `ZBS_PLUGIN_DIR` since the file that defines it hasn't been required at this point.
AFAIK then we can't rely on other types of e.g. WP constants like `WP_PLUGIN_DIR` or `ABSPATH` as well because that would assume we know where the plugin is located - although they should exist because we run `\WorDBless\Load::load()`.

Because the tests lives "inside" the CRM directory, then AFAIK this is probably the most robust way to bootstrap it; like when we require the autoloader:

```php
require_once __DIR__ . '/../../vendor/autoload.php';
``` 

>Note that this won't always work depending on the setup

What specific setup(s) do you have in mind where this would fail?